cli/waiter/cli.py

```diff
@@ -0,0 +1,93 @@
+import argparse
+import logging
+from urllib.parse import urlparse
+
+from waiter import configuration, metrics, http, version
+from waiter.subcommands import create, show
+
+parser = argparse.ArgumentParser(description='waiter is the Waiter CLI')
```

Comment:
Nitpick: lexicographic order in the arguments below

Reply:
I actually think it's nice to have `--cluster` and `--url` next to each other (without `--config` in between them) since they serve a similar purpose